src/server/cloak/index.js

```diff
@@ -90,7 +89,16 @@ module.exports = (function() {
         delete config.express;
       }
 
-      io.set('log level', config.logLevel);
+      // Cloak defaults Socket.IO log level to 1
+      io.set('log level', 1);
```

Comment:
This is probably obvious from my review, but my advice would be

- Accept the few additional lines of cruft in favor of supporting your existing users
- Or:
  - If you have specific plans for additional breaking changes, document them in a "version 1.0 roadmap".
  - Otherwise, release 1.0 . A pre-1.0 library (especially one maintained in this way) is not attractive for live deployment.